pom.xml

```diff
@@ -516,6 +516,15 @@
     </build>
 
     <repositories>
+        <repository>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <id>kogito</id>
+            <name>kogito</name>
+            <url>https://repository.jboss.org/nexus/content/groups/kogito-public/</url>
+        </repository>
```

Comment:
Is this really necessary?

Reply:
That's a draft @ppalaga It's still served from a staging repo. We won't merge this.

Reply:
removed